daprdocs/content/en/reference/cli/dapr-init.md

```diff
@@ -33,11 +33,14 @@ dapr init [flags]
 | `--namespace`, `-n`  |                      | `dapr-system` | The Kubernetes namespace to install Dapr in                                          |
 | `--runtime-version`  |                      | `latest`      | The version of the Dapr runtime to install, for example: `1.0.0`                     |
 | `--slim`, `-s`       |                      | `false`       | Exclude placement service, Redis and Zipkin containers from self-hosted installation |
-
+| `--image-registry`   |                      |               | Pulls container images required by Dapr from the given image registry                    |
+|                      |DAPR_DEFAULT_IMAGE_REGISTRY|          | In standalone mode, it is used to specify the default container registry to pull images from. When its value is set to `GHCR` or `ghcr` it pulls the required images from Github container registry. To default to Docker hub as default, just unset this env variable.|
```

Comment:
This DAPR_DEFAULT_IMAGE_REGISTRY env var should be in the --image-registry row, not its own row, since it is effectvively the same way to do the setting.